pkg/agent/controller/ipseccertificate/ipsec_certificate_controller_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build linux
+// +build linux
```

Comment:
Why need this?

Reply:
ipsec is not supported on Windows. Without this mark, I hit failures in Windows unit test as the case would run on Windows.

Reply:
Is this an existing issue and not related to this PR?

Reply:
I hit it in this PR, but nothing changed in the related logic.

Reply:
That's strange. Maybe we need more investigation.
@tnqn Any idea?

Reply:
was it the issue https://github.com/antrea-io/antrea/pull/3852 fixed? Even though the feature itself doesn't work on Windows platform, we could run its unit test on the unsupported paltforms if the code itself doesn't have platform dependency, otherwise no much unit tests would run on MacOS since antrea-agent doesn't support MacOS node, which is used by many developers.

Reply:
I don't think it is the same issue as https://github.com/antrea-io/antrea/pull/3852 . I just rebased the code yesterday, and hit the failure after rebase. 
If you don't think we should ignore those unit tests on other platforms, I would remove this mark. And I could create a new issue to track it if the failure is hit again.

Reply:
Yes, better to create a issue to track it if the failure is hit again, it doesn't seem related to windows.